### DISCUSSION TITLE
`Boost`协议兼容

### DISCUSSION THREAD

**Discussion by Fatman13** [filecoin-project/venus#5495] [Ideas]
## 背景

自`fil-singapore`和 @Lex 讨论了一系列 Boost mk1.2 协议兼容的相关事宜之后，`venus`也开始有条不紊地推进 mk1.2 协议兼容工作。本讨论收集当前相关的进度以及资料。

## 进度

- [ ] [Slack](https://filecoinproject.slack.com/archives/CEHHJNJS3/p1667284673948379?thread_ts=1664334507.999999&cid=CEHHJNJS3)合作意向讨论
- [ ] Boost协议代码[解读](https://venus.filecoin.io/zh/research/boost-protocol-research.html#%E7%BB%93%E8%AE%BA)
- [ ] Boost协议`mk1.2`兼容[设计](https://venus.filecoin.io/zh/research/impl-mk-1.2.0.html)

## 其他

- [ ] lotus-market EOL 2023/01/31 之后[不再维护](https://filecoinproject.slack.com/archives/C027TQMUVJN/p1668756570528819)

**Comment by brendalee** [comment]
any update on this? :)

**Reply by Fatman13** [reply]
CC @hunjixin

**Reply by Fatman13** [reply]
Hello, @brendalee, do you think we could push for [multi SP support](https://github.com/filecoin-project/boost/discussions/1010) into the new protocol so that Venus could offer better UX for its user base?
